src/liblsquic/lsquic_di_hash.c

```diff
@@ -53,15 +53,14 @@ struct data_block
     unsigned char           db_data[DB_DATA_SIZE];
 };
 
-typedef char db_set_covers_all_db_data[(N_DB_SETS * 64 >= DB_DATA_SIZE) - 1];
-typedef char db_set_no_waste[(N_DB_SETS * 64 - 64 <= DB_DATA_SIZE) - 1];
-typedef char db_block_is_4K[(sizeof(struct data_block) == 0x1000) - 1];
+typedef char db_set_covers_all_db_data[(N_DB_SETS * 64 >= DB_DATA_SIZE) ?1: - 1];
+typedef char db_set_no_waste[(N_DB_SETS * 64 - 64 <= DB_DATA_SIZE)?1: - 1];
+typedef char db_block_is_4K[(sizeof(struct data_block) == 0x1000) ?1:- 1];
```

Comment:
These changes avoid arrays of size 0, which is not allowed by the Windows compiler.  I recommend they remain.